lighthouse-core/gather/gather-runner.js

```diff
@@ -494,6 +494,41 @@ class GatherRunner {
     };
   }
 
+  /**
+   * Populates the important base artifacts from a fully loaded test page.
```

Comment:
I think this jsdoc description is off ;)

Reply:
hehe